internal/server/singleprocess/service_entrypoint.go

```diff
@@ -208,11 +208,24 @@ func (s *service) EntrypointLogStream(
 			// Read our instance record
 			instance, err := s.state.InstanceById(batch.InstanceId)
 			if err != nil {
-				return err
+				if status.Code(err) == codes.NotFound {
```

Comment:
Can we add some logging around this so we can look in the logs to see whats happening?

Reply:
Done!